src/lib.rs

```diff
@@ -116,13 +116,12 @@ where
     ///
     /// This runs in an infinite loop that should be terminating once the other end of the socket
     /// (the VMM) disconnects.
+    // TODO: current implementation is a little wired, the `listener` could only accept one
```

Comment:
s/wired/weird/ ?. But perhaps we should state something like "the current implementation has a couple of limitations. The (...)" instead.

Reply:
Thanks, that's a strong marker for me to improve the code, but the comment has bad smells. So changed to
```
// TODO: the current implementation has limitations that only one incoming connection will be
    // handled from the listener. Should it be enhanced to support reconnection?
```